### ISSUE TITLE
Whats the minimum to catch rare fish and whats the difference between each of the rods

### ISSUE THREAD

**Issue by Peant8** [Draylar/go-fish#21]
i just wanna know because i can only seem to catch cod even with a diamond rod

**Comment by Draylar** [comment]
There is no minimum, but you should be catching modded fish as much as you catch vanilla fish. Where are you fishing? Do you have any other fishing mods installed, such as  Netherite Plus?

**Comment by Peant8** [comment]
i didnt i was just fishing in the ocean on creative maybe i was just unlucky

**Comment by Peant8** [comment]
what is the difference between each of the rods btw?

**Comment by Windyxelor** [comment]
I can't get any of the modded fish with a diamond rod, I do have Netherite Plus downloaded. It might be because I'm on 1.16.5 and the version of this mod I'm using is 1.16.4, but I dunno. Please help

**Comment by Windyxelor** [comment]
I can't get any of the modded fish or crates with the diamond rod or any others, I do have Netherite Plus installed and I'm fishing in a Plains biome, help

**Comment by Draylar** [comment]
Have you tried Go Fish without Netherite Plus?

**Comment by Windyxelor** [comment]
Yes, I fished 10 times with the diamond rod in a plains biome and got no
modded fish
